### ISSUE TITLE
Accepting a task

### ISSUE THREAD

**Issue by amatefi** [paulfilkin/tradosClarity#18]
- When the screen reader moves through the columns, the scroll bar does not follow, making it difficult to track the focus.
- There is a table with 18 columns containing project details. The last column, which includes the download option, is unlabeled. To accept a task, navigate to the first column corresponding to the desired project name and press the Space bar twice to select it.
- When the screen reader moves through the columns, the horizontal scroll bar does not follow, making it difficult to keep track of the focus.
- Project tab: You will see the Translation task displayed  select it by pressing the space bar on its name. Note that the screen reader does not announce the checkbox or confirm that the translation document has been selected.
